ccm_web/server/src/api/api.service.ts

```diff
@@ -21,4 +29,44 @@ export class APIService {
       }
     }
   }
+
+  async getCourseName (userLoginId: string, courseId: number): Promise<string | null> {
+    const requestor = await this.canvasService.createRequestorForUser(userLoginId)
+    let name = null
+    try {
+      const response = await requestor.get<Course>(`courses/${courseId}`)
```

Comment:
is the library is throwing an error if the error is other than `2xx`?

Reply:
Yes, but also `3xx` I guess. See https://github.com/sindresorhus/got#gothttperror

We can customize this if we want, using the `throwHTTPErrors` option: https://github.com/sindresorhus/got#throwhttperrors

Maybe I need to modify the unusual code check. Right now I'm using `error instanceof HTTPError && error.response.statusCode !== HttpStatus.OK`, but maybe just `error instanceof HTTPError` is sufficient.

Reply:
I decided to remove the check for the non-`200` status code; see here https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/121/commits/d684eeb2fca3bd59040f59df42dcfda583dbd18d.

This might be a moot point, since I don't think we'll encounter `2XX` or `3XX` other than `200`, but I guess those are still technically `OK` response codes.